libarchive/archive_cryptor.c

```diff
@@ -366,6 +366,7 @@ aes_ctr_update(archive_crypto_ctx *ctx, const uint8_t * const in,
 	(void)in_len; /* UNUSED */
 	(void)out; /* UNUSED */
 	(void)out_len; /* UNUSED */
+        aes_ctr_encrypt_counter(ctx); /* UNUSED */
```

Comment:
Please put the aes_ctr_encrypt_counter fix in a separate pull request (it is easier to keep track of changes that way).